src/framework/global/internal/globalconfiguration.cpp

```diff
@@ -74,7 +74,7 @@ QString GlobalConfiguration::resolveAppDataPath() const
 #endif
 
 #ifdef Q_OS_WIN
-    QDir dir(QCoreApplication::applicationDirPath() + QString("/../" MUSE_APP_INSTALL_NAME));
+    QDir dir(QCoreApplication::applicationDirPath() + QString("/../"));
```

Comment:
It was like that before my changes, I just renamed it.
```
QDir dir(QCoreApplication::applicationDirPath() + QString("/../" MUSESCORE_INSTALL_NAME));
```

And also keep for other platforms
```
QDir dir(QCoreApplication::applicationDirPath() + QString("/../share/" MUSESCORE_INSTALL_NAME));
```
https://github.com/musescore/MuseScore/blob/4.3.0/src/framework/global/internal/globalconfiguration.cpp#L68

Have you done the install step? Without it, the tools were always not displayed

Reply:
Yes, of course I did the install step (but yes, such stupid mistakes are possible ;-)). But `MUSESCORE_INSTALL_NAME` is not the same as `MUSE_APP_INSTALL_NAME`, the latter contains "App", but no such directory exists on Windows

Reply:
And I'm not saying that my change is correct, just that it fixes the issue, on Windows.

Reply:
Hmm, ok, thanks, I'll merge this fix and later I'll see what actually happened